packages/block-library/src/post-title/index.php

```diff
@@ -29,7 +29,7 @@ function render_block_core_post_title( $attributes, $content, $block ) {
 
 	$title = get_the_title( $post_ID );
 	if ( isset( $attributes['isLink'] ) && $attributes['isLink'] ) {
-		$title = sprintf( '<a href="%1s" target="%2s" rel="%3s">%4s</a>', get_the_permalink( $post_ID ), $attributes['linkTarget'], $attributes['rel'], $title );
+		$title = sprintf( '<a href="%1$s" target="%2$s" rel="%3$s">%4$s</a>', get_the_permalink( $post_ID ), $attributes['linkTarget'], $attributes['rel'], $title );
```

Comment:
Isn't this needed only if we reuse some arguments? If we use them just once (like here) and the order is correct the `$` can be omitted, no?

Reply:
It's usually good practice to use them anyway, to avoid bugs in the future. This code was already using them, but with an incorrect format - `%1s`, the `$` was missing.

Reply:
That what I meant - that the `$` was not needed here. The `%` is needed and was there. Having said that I have no strong opinions on this one 😄

Reply:
I think there was WPCS rule for it, but can find it right now 😅